easybuild/easyconfigs/m/M1QN3/M1QN3-3.3-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CmdCp'
+
+name = 'M1QN3'
+version = '3.3'
+
+homepage = 'https://who.rocq.inria.fr/Jean-Charles.Gilbert/modulopt/optimization-routines/m1qn3/m1qn3.html'
+description = "A solver of large-scale unconstrained minimization problems"
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
```

Comment:
Is this something that we would also want to install with intel compilers?

Reply:
Good point, it's Fortran, so yes...

I'll move this to `GCC` toolchain

Reply:
fixed in caf2012